Mage/src/main/java/mage/abilities/effects/keyword/FatesealEffect.java

```diff
@@ -54,7 +54,7 @@ public boolean apply(Game game, Ability source) {
                     return true;
                 }
                 for (int i = 0; i < count; i++) {
-                    Card card = opponent.getLibrary().removeFromTop(game);
+                    Card card = opponent.getLibrary().getFromTop(game);
```

Comment:
And don’t forget to add cards removing in code like that here. Maybe it’s better to keep removeFromXXX methods.